include/coreinit/filesystem_fsa.h

```diff
@@ -593,7 +607,8 @@ struct FSARequest {
         FSARequestUnmount unmount;
         FSARequestUnmountWithProcess unmountWithProcess;
         FSARequestWriteFile writeFile;
-        WUT_UNKNOWN_BYTES(0x51C);
+        FSARequestChangeOwner changeOwner;
+        WUT_UNKNOWN_BYTES(0x28C);
```

Comment:
This is padding for the union and should stay as `0x51C`.

Reply:
that would make this struct bigger, no?

Reply:
It's a union not a struct.

Reply:
my mistake, i scrolled up way to high and saw the struct but didn't see the union.